src/index.ts

```diff
@@ -36,6 +36,11 @@ export type SerializationOptions = {
    * More details here: https://buildkite.com/docs/pipelines/dependencies#defining-explicit-dependencies
    */
   explicitDependencies?: boolean;
+  /**
+   * Whether accept all conditional steps regardlss of it's being rejected or not. This is particularlly helpful in some places whether you want to generate the entire
+   * graph without conditions. Plese note this would only work when `explicitDependencies` above is set to true.
```

Comment:
```suggestion
   * Whether to accept all conditional steps regardless of what their `.accept()` function would return. This is particularly helpful in some places whether you want to generate the entire
   * graph without pruning.
```

- [ ] Use the type system to only allow `acceptAllConditions` in conjunction with `explicitDependencies: true`

Reply:
just read further below, I think you did this already.